package.json

```diff
@@ -41,12 +41,11 @@
     "chokidar-cli": "^1.2.0",
     "jade": "^1.11.0",
     "live-server": "^1.0.0",
-    "mkdirp": "^0.5.1",
     "mofo-style": "^2.3.0",
-    "ncp": "^2.0.0",
     "node-sass": "^3.4.2",
     "npm-run-all": "^1.5.1",
     "sass-lint": "^1.5.0",
-    "shelljs": "^0.7.0"
+    "shelljs": "^0.7.0",
```

Comment:
It's for the deploy script. I usually don't lock down devdeps, but we could do that if you feel like it's a good idea?